ignite/_utils.py

```diff
@@ -2,6 +2,10 @@
 # For compatibilty
 from ignite.utils import convert_tensor, apply_to_tensor, apply_to_type, to_onehot
 
+__all__ = [
```

Comment:
This one is not a public API, we can ignore it ?

Reply:
I'll remove it